extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -75,6 +76,13 @@
     @ConfigItem(defaultValueDocumentation = "import.sql in DEV, TEST ; no-file otherwise")
     public Optional<String> sqlLoadScript;
 
+    /**
+     * The name of the charset used by the schema generation resource. Without specifying this configuration property, the JVM
+     * default charset is used.
+     */
+    @ConfigItem(defaultValueDocumentation = "JVM default charset")
+    public Optional<Charset> ddlCharset;
```

Comment:
Hello @geoand, perhaps this should just be defaulted to `UTF-8` to remove the inconsistency of system level defaults? AFAIK, Java (in recent versions) too has started defaulting to `UTF-8` for some of its IO APIs to avoid this system level inconsistencies.

Reply:
I think that's reasonable, but I didn't because I wanted to keep it consistent with the other charset property.

If  @Sanne and @gsmet also think it should be UTF-8, I'll gladly update it.

Reply:
My inclination would be to go with UTF-8 by default too.

I don't like the name though because it's not the DDL charset, it's the charset used for schema generation and the import scripts (or database initialization in general).

Reply:
Removed this completely